README.md

```diff
@@ -1 +1,24 @@
-## Hello World Java
\ No newline at end of file
+# Codio Java Project Template
+This starterpack serves as a template for Java projects in Codio.
+
+## Open HelloWorld.java via file tree
+Click on the HelloWorld.java file in the file tree to open it in the code editor.
+
+You can right-click in the file tree to rename this file or delete it and start from a new file (see below).
+
+## Compile, Run or Both
+Use the Run button (that looks like a Rocketship) to Compile and Run the file your curser is in.
+
+![RunMenuJava](RunMenuJava.png)
```

Comment:
This line should link to a URL

Reply:
the link is https://global.codio.com/platform/readme.resources/RunMenuJava.png